org-cyf-guides/content/contributing/minutes/index.md

```diff
@@ -17,6 +17,56 @@ Attendees:
 
 ---
 
+## 2025-01-14
+
+Attendees: Ali, Mitch, Sally
+
+### ❗Actions from last time
+
+* [ ] Mitch (carry-over): Write up a ticket to replace Jest with vanilla Node testing libraries
+* [ ] Mitch: Write up why the curriculum looks like it does, and isn't set up like a reference
+* [ ] Ali: Write up a ticket for "Make it easy to say out loud here's what we teach and why"
+
+### 📝 Agenda points
+
+#### Piscine projects
+
+* Daniel: We should pick the actual Piscine projects for February this week. I'm not sure we actually have two team projects right now? Days Calendar is one, so we should probably do that. Repetition Tracker is probably team-able-enough, so maybe that? And as an individual project either Shared Bookmarks or Music Data or Codewars Leaderboard? We should decide this this week so we can update the site
+* Decision: agree with Daniel's proposal above. Days Calendar & Repetition Tracker are the team projects. Shared Bookmarks, Music Data & Codewars Leaderboard can be individual projects
```

Comment:
Do we want to pick one individual project, or allow people choice?

@CodeYourFuture/piscine-team

Reply:
I don't think we specifically discussed this. I would be open to the idea of giving a choice. The major downside in my mind is that we find out during the course that one is much harder/easier than the others. But on the other hand, there is only one way to found out?

Reply:
Cool, I'll send out a PR :) Thanks!